src/go/k8s/pkg/resources/resource_integration_test.go

```diff
@@ -110,6 +111,17 @@ func TestEnsure_ConfigMap(t *testing.T) {
 	cluster = cluster.DeepCopy()
 	cluster.Name = "ensure-integration-cm-cluster"
 
+	secret := corev1.Secret{
```

Comment:
can you help me understand this addition? it just creates secret and that's it? 🤔 is it needed for something?

Reply:
It's needed for unit testing because I enabled cloud storage